src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -305,6 +315,9 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
   allOk = false
   snackbar = false
   snackbarText = ''
+  errorDialog = false
+  errorDialogText = undefined as string
+  errorDialogTitle = undefined as string
```

Comment:
If I used "null" here, the dialog didn't use its default prop values. This works.